spec/node-spec.js

```diff
@@ -240,6 +240,24 @@ describe('node feature', () => {
       child.stdout.on('data', outDataHandler)
     })
 
+    it('does not start the v8 inspector when --inspect is after a -- argument', (done) => {
+      child = ChildProcess.spawn(remote.process.execPath, [path.join(__dirname, 'fixtures', 'module', 'run-as-node.js'), '--', '--inspect'])
+
+      let output = ''
+      function dataListener (data) {
+        output += data
+      }
+      child.stderr.on('data', dataListener)
+      child.stdout.on('data', dataListener)
+      child.on('exit', () => {
+        if (output.trim().startsWith('Debugger listening on ws://')) {
+          done(new Error('Inspector was started when it should not have been'))
+        } else {
+          done()
+        }
+      })
+    })
```

Comment:
I'd like to see a test for the attack vector, rather than the mitigation. This is only a security issue if untrusted data can make it into the argument list.

Reply:
This isn't testing the mitigation, this issue explicitly only occurred when you were using the "--" mitigation for a different issue.  This tests the repro for **this** issue